[Bug]: chinadns-ng failed to build

### 描述您遇到的bug

编译时chinadns-ng报编译失败

### 复现此Bug的步骤

183 make[3] -C feeds/packages/utils/bc compile
184 make[3] -C feeds/packages/admin/htop compile
185 make[3] -C feeds/packages/utils/nano compile
186 make[3] -C feeds/packages/utils/pciutils compile
187 make[3] -C package/network/utils/ipset compile
188 make[3] -C feeds/packages/utils/usbutils compile
189 make[3] -C feeds/packages/utils/vim compile
190 make[3] -C feeds/packages/utils/zsh compile
191 make[3] -C package/kernel/button-hotplug compile
192 make [3] -C package/lean/autocore compile
193 make[3] -C package/network/ipv6/odhcp6c compile
194 make[3] -C package/lean/r8152 compile
195 make[3] -C package/openwrt-passwall/chinadns-ng compile
196 ERROR: package/openwrt-passwall/chinadns-ng failed to build.
197 make -r world: build failed. Please re-run make with -j1 V=s or V=sc for a higher verbosity level to see what's going on
198 make: *** [/home/runner/work/Openwrt/Openwrt/openwrt/include/toplevel.mk: 231: world] Error 1
199 make[2]: Entering directory '/home/runner/work/Openwrt/Openwrt/openwrt/scripts/config'
200 make[2]: 'conf' is up to date.
201 make[2]: Leaving directory '/home/runner/work/Openwrt/Openhrt/openwrt/scripts/config*

### 您想要实现的目的

期待编译通过

### 日志信息

183 make[3] -C feeds/packages/utils/bc compile
184 make[3] -C feeds/packages/admin/htop compile
185 make[3] -C feeds/packages/utils/nano compile
186 make[3] -C feeds/packages/utils/pciutils compile
187 make[3] -C package/network/utils/ipset compile
188 make[3] -C feeds/packages/utils/usbutils compile
189 make[3] -C feeds/packages/utils/vim compile
190 make[3] -C feeds/packages/utils/zsh compile
191 make[3] -C package/kernel/button-hotplug compile
192 make [3] -C package/lean/autocore compile
193 make[3] -C package/network/ipv6/odhcp6c compile
194 make[3] -C package/lean/r8152 compile
195 make[3] -C package/openwrt-passwall/chinadns-ng compile
196 ERROR: package/openwrt-passwall/chinadns-ng failed to build.
197 make -r world: build failed. Please re-run make with -j1 V=s or V=sc for a higher verbosity level to see what's going on
198 make: *** [/home/runner/work/Openwrt/Openwrt/openwrt/include/toplevel.mk: 231: world] Error 1
199 make[2]: Entering directory '/home/runner/work/Openwrt/Openwrt/openwrt/scripts/config'
200 make[2]: 'conf' is up to date.
201 make[2]: Leaving directory '/home/runner/work/Openwrt/Openhrt/openwrt/scripts/config*

### 截图

_No response_

### 系统相关信息

passwall 4.76-1

### 其他信息

_No response_

--- a/chinadns-ng/Makefile
+++ b/chinadns-ng/Makefile
@@ -9,28 +9,28 @@ UNPACK_CMD=$(CP) $(DL_DIR)/$(PKG_SOURCE) $(PKG_BUILD_DIR)/chinadns-ng
 
 ifeq ($(ARCH),aarch64)
   PKG_SOURCE:=chinadns-ng@aarch64-linux-musl@generic+v8a@fast+lto
-  PKG_HASH:=0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5
+  PKG_HASH:=cd53686626f678fff3122b61aabee6c3bf9dad7e7577a8efc5f432fdeaf8d975
 else ifeq ($(ARCH),arm)
   ARM_CPU_FEATURES:=$(word 2,$(subst +,$(space),$(call qstrip,$(CONFIG_CPU_TYPE))))
   ifeq ($(ARM_CPU_FEATURES),)
     PKG_SOURCE:=chinadns-ng@arm-linux-musleabi@generic+v7a@fast+lto
-    PKG_HASH:=0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5
+    PKG_HASH:=106a53e63991a78631c1ca6bfc648600bb9cabe08045b443acaaaf3113401b00
   else
     PKG_SOURCE:=chinadns-ng@arm-linux-musleabihf@generic+v7a@fast+lto
-    PKG_HASH:=0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5
+    PKG_HASH:=a04997e353732a77a374b869f8b37f5e7a3451e5659b8cb3db1e31f40f0a06d7
   endif
 else ifeq ($(ARCH),mips)
   PKG_SOURCE:=chinadns-ng@mips-linux-musl@mips32+soft_float@fast+lto
-  PKG_HASH:=0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5
+  PKG_HASH:=18970e6191661c3f2398a18ce8b003758c6a6c0f3c5d164a566fb108ab5bc1da
 else ifeq ($(ARCH),mipsel)
   PKG_SOURCE:=chinadns-ng@mipsel-linux-musl@mips32+soft_float@fast+lto
-  PKG_HASH:=0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5
+  PKG_HASH:=26aa029b141a3d25e078c4b92c06c93a87c06b0516e86c13d32a9ee865dd2ca2
 else ifeq ($(ARCH),i386)
   PKG_SOURCE:=chinadns-ng@i386-linux-musl@i686@fast+lto
-  PKG_HASH:=0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5
+  PKG_HASH:=4147e3be49f5ef5c0200b6c62cb8036e7944b4f0729ede83faef33c828d6ff91
 else ifeq ($(ARCH),x86_64)
   PKG_SOURCE:=chinadns-ng@x86_64-linux-musl@x86_64@fast+lto
-  PKG_HASH:=0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5
+  PKG_HASH:=ea17998c450eaf08cf82b8b2bc3767bd39f0d7686a53e1c5ad972b48db750974
 else
   PKG_SOURCE:=dummy
   PKG_HASH:=dummy

